app/utils/filename_validation.py

```diff
@@ -0,0 +1,50 @@
+
+ALLOWED_EXTENSIONS = {'png', 'jpg', 'jpeg', 'gif'}
+
+def allowed_file(filename):
+    """
+    Check the extension of the file and ensure it's a png,
+    jpg, jpeg or gif file.
+    """
+    return '.' in filename and \
+           filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+"""
+    The functions below are necessary to transform the
+    upload image filename to a valid one.
+    When an image filename containing brackets and blank spaces is uploaded,
+    it is saved in the "uploads" folder without round brackets and 
+    with underscores replacing the blank spaces.
+"""
+def rm_string_spaces(string):
+    """
+    It replaces every blank space in the string with an underscore.
+
+    """
+    list_splits = string.split(" ")
```

Comment:
I think for this you can use the `str.replace()` of the string type.

Reply:
done